packages/server/src/services/email-sending.service.ts

```diff
@@ -0,0 +1,45 @@
+import nodemailer from 'nodemailer';
+
+import { Pages } from 'core/constants/pages';
+import { QueryParams } from 'core/constants/query-params';
+
+export default class EmailSendingService {
+  public async sendEmail(mailOptions: Record<string, unknown>): Promise<void> {
+    const {
+      EMAIL_HOST,
+      EMAIL_PORT,
+      EMAIL_USER,
+      EMAIL_PASS
+    } = process.env;
+
+    const smtpTransport = nodemailer.createTransport({
+      host: EMAIL_HOST,
+      port: EMAIL_PORT,
+      auth: {
+        user: EMAIL_USER,
+        pass: EMAIL_PASS
+      }
+    });
+
+    try {
+      await smtpTransport.sendMail(mailOptions);
+    } catch (error) {
+      console.log(error);
+    }
+  }
+
+  public async sendActivationMessage(userEmail: string, host: string, emailVerificationToken: string): Promise<void> {
+    const link = `${host}${Pages.EMAIL_CONFIRMATION_PAGE}?${QueryParams.EMAIL_VERIFICATION_TOKEN}=${emailVerificationToken}`;
+    const mailOptions = {
+      to: userEmail,
+      subject: 'Please confirm your email account',
+      html: `
+        Hello,<br> Please Click on the link to verify your email.
```

Comment:
I'd suggest to add name into email

Reply:
for now, we don't have such a field. will add in the scope of future PRs.

Reply:
I would also suggest to move text-creator into another entity

Reply:
Use name for now, and update after a new field will be introduced